File: remote/src/main/java/com/kelsos/mbrc/domain/Album.java
package com.kelsos.mbrc.domain;

public class Album {
  private long id;
  private String name;
  private String artist;
  private String cover;
  private String year;

  public Album(long id, String name, String artist, String cover, String year) {
    this.id = id;
    this.name = name;
    this.artist = artist;
    this.cover = cover;
    this.year = year;
  }

  public String getCover() {
    return cover;
  }

  public String getArtist() {
    return artist;
  }

  public String getName() {
    return name;
  }

  public long getId() {
    return id;
  }

  public String getYear() {
    return year;
  }
}

File: app/src/main/kotlin/com/kelsos/mbrc/domain/Album.kt
package com.kelsos.mbrc.domain

import com.kelsos.mbrc.data.db.RemoteDatabase
import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.QueryModel
import com.raizlabs.android.dbflow.structure.BaseQueryModel

@QueryModel(database = RemoteDatabase::class)
class Album() : BaseQueryModel() {
  @Column var id: Long = 0
  @Column var name: String = ""
  @Column var artist: String = ""
  @Column var cover: String = ""
  @Column var year: String = ""
}
